.github/workflows/build.yml

```diff
@@ -101,6 +169,6 @@ jobs:
           TF_VAR_publish_bucket:   ${{ vars.PUBLISH_BUCKET }}
         run: |
           echo "${{ secrets.JPLICA_CERT }}" >> ${{ runner.temp }}/JPLICA.pem
- 
-          source bin/config.sh sit
-          terraform apply -auto-approve
+
+          source bin/config.sh ${{ vars.TF_VENUE }}
```

Comment:
Doesn't this need to be lowercased like here:
https://github.com/podaac/swodlr-user-notify/blob/682e19d2c8aaee8143fa8d24727698b00b32d33b/.github/workflows/build.yml#L169

Reply:
It seems to not need that for some reason, but I think it's a good idea to apply the lowercase to everything